api/src/main/java/uk/gov/hmcts/bar/api/data/service/PaymentInstructionService.java

```diff
@@ -213,22 +209,34 @@ public MultiMap getPaymentStatsByUserGroupByType(String userId, String status, b
 
         MultiMap paymentInstructionStatsGroupedByBgc = new MultiValueMap();
         results.stream().forEach(stat -> {
-            Link detailslink = linkTo(methodOn(PaymentInstructionController.class)
-                .getPaymentInstructionsByIdamId(userId, status,
-                    null, null, null, null, null,
-                    null, null, null, stat.getPaymentType(), null, null, stat.getBgc())
-            ).withRel(STAT_DETAILS);
+            Link detailslink = null;
+
+            try {
+                detailslink = linkTo(methodOn(PaymentInstructionController.class)
+                    .getPaymentInstructionsByIdamId(userId, status,
+                        null, null, null, null, null,
+                        null, null, null, stat.getPaymentType(), null, null, stat.getBgc())
+                ).withRel(STAT_DETAILS);
+            } catch (BarUserNotFoundException e) {
```

Comment:
it user doesn't exist then return empty results as there won't be any records for that user.